src/manage.py

```diff
@@ -468,12 +468,32 @@ def update_element_tags(self):
     self.library.tags_updated = datetime.datetime.now()
     return tag_map
 
+  def update_package_tags(self):
```

Comment:
Is this part of a pipeline e.g. on a cron job? Or is it part of some synchronous user workflow?